src/vchan_cmds.c

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright (c) 2024 EPAM Systems
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <domain.h>
+#include <vch.h>
+#include <zephyr/kernel.h>
+#include <zephyr/logging/log.h>
+#include <zephyr/shell/shell.h>
+
+LOG_MODULE_REGISTER(vchan_cmd);
+
+#define ECHO_SERVER_THREAD_PRIO    7
+static struct k_thread echo_thread;
+static struct k_thread *pecho_thread = NULL;
+struct vch_handle handle;
+
+static K_THREAD_STACK_DEFINE(main_stack, 8192);
+static volatile int vchan_server_stop = 0;
+static char buf[10];
+#define PATH_TEMPLATE "/local/domain/%d/echo"
+char path[30];
+
+static void vchan_echo_server(void *p1, void *p2, void *p3)
+{
+	struct vch_handle *h = (struct vch_handle *)p1;
+	int size;
+	int count = 0;
+
+	ARG_UNUSED(p2);
+	ARG_UNUSED(p3);
+	printk("Thread started\n");
```

Comment:
Please, use Zephyr logging subsystem instead of printk, you've already registered logging module on line 13

Reply:
Fixed